.github/workflows/CI.yml

```diff
@@ -83,6 +84,13 @@ jobs:
         make clean   # make sure we only see installation artifacts
         make test_install
 
+    - name: Design Edit Test
+      if: matrix.mode == 'designEdit'
+      run: |
+        make
```

Comment:
Looks like you are missing a "make clean" , check the target above line 74

Reply:
CI is fixed. Please merge.